terraform/gke.tf

```diff
@@ -59,14 +60,15 @@ module "gke" {
     # Add the local IP of the workstation that applies the Terraform to authorized networks
     {
       display_name : "Local IP",
-      cidr_block : "${chomp(data.http.installation_workstation_ip.body)}/32"
+      cidr_block : "${chomp(data.http.installation_workstation_ip.response_body)}/32"
     }
   ]
 
   node_pools = [for tenant_name, config in local.tenants : {
     auto_upgrade                = true
     enable_integrity_monitoring = true
     enable_secure_boot          = true
+    enable_shielded_nodes       = true
```

Comment:
As we mentioned, this is something we already enable at cluster level. Remove.

Reply:
Done